### PR TITLE
[CROSSDATA-6] [DOC] Configuration document

### PR DESCRIPTION
**Pull request by jjlopezm**
Describe what and how to configurate driver, servers and catalog of Crossdata

### CODE REVIEW

File: doc/src/site/sphinx/3_configuration.rst [hunk 1]
```diff
@@ -1,8 +1,97 @@
...
+ - crossdata.catalog.mysql.db.name: Database name in the database catalog.
+
+ - crossdata.catalog.mysql.db.tableName: Table name where the metadata will be stored in the database catalog
+
+ - crossdata.catalog.mysql.db.user: Allowed user to connect using JDBC to the catalog database.
+
+ - crossdata.catalog.mysql.db.pass: Password to connect using JDBC to the catalog database.
+
+Here is an example of ths configuration file::
+
```
Comment (ccaballe) [inline doc/src/site/sphinx/3_configuration.rst]: typo error: "
+Here is an example of **ths** configuration file"

**Comment by ccaballe** [comment]
+1

**Comment by luismcl** [comment]
+1

**Comment by miguel0afd** [comment]
LGTM
+1

**Pull request merged**